src/blogs/performance.md

```diff
@@ -0,0 +1,71 @@
+---
+
+title: Monitoring DreamHack - the World's Largest Digital Festival
+author: rahul
+author_info: Software Engineer at MayaData, working on Cloud Native Tech
+tags: devops
+image: img1.png
+avatar: captain.png
```

Comment:
can we skip this and get avatar image src with author's name?

Reply:
updated it and fetch images from author names